manifests/sqlitebrowser/sqlitebrowser/3.11.2.yaml

```
@@ -0,0 +1,17 @@
+Id: sqlitebrowser.sqlitebrowser
```

```suggestion
Id: Sqlitebrowser.Sqlitebrowser
```
or
```suggestion
Id: SQLiteBrowser.SQLiteBrowser
```

Packages use PascalCasing. Not sure about the second suggestion because I can't find good example of the casing used here - but SQLite is cased like that. Your choice.